app/models/aliquot/remover.rb

```diff
@@ -42,6 +45,9 @@ def remove_downstream_aliquots
   end
 
   def on_downstream_aliquots(aliquots_to_remove)
+    transfer_requests_as_source.each do |request|
+      request.target_asset.process_aliquots(aliquots_to_remove)
+    end
```

Comment:
Bit of duplication here, but one of the few places where requests and transfer requests get treated the same.